backend/map_backend/management/commands/load_data.py

```diff
@@ -0,0 +1,108 @@
+from django.core.management.base import BaseCommand, CommandError
+from map_backend.models import Course, Calculator, Program, RequirementGroup, RequirementItem, CourseList
+from django.conf import settings
+import os, json
+
+def load_data(courses_path, programs_path):
```

Comment:
Rewriting all these management commands will break the current mosaic api stuff that we have right now, since that produces those 4 json files which the old management commands used. If this was discussed with everyone and they think that's fine, it's good with me. But it will require a bit of a rewrite of the current mosaic api stuff to get it working with this new management command.